### PR TITLE
keepassxc: update to 2.4.0.

### PR DESCRIPTION
**Pull request by gspe**
Update to 2.4.0 and simplified template

### CODE REVIEW

File: srcpkgs/keepassxc/template [hunk 1]
```diff
@@ -1,45 +1,28 @@
...
 post_install() {
-	vlicense COPYING
-	vlicense LICENSE.BOOST-1.0
-	vlicense LICENSE.BSD
-	vlicense LICENSE.CC0
-	vlicense LICENSE.NOKIA-LGPL-EXCEPTION
-	vlicense LICENSE.LGPL-3
-	vlicense LICENSE.LGPL-2.1
-	vlicense LICENSE.GPL-2
+ vlicense COPYING
```
Comment (maxice8) [inline srcpkgs/keepassxc/template]: wrong indent

File: srcpkgs/keepassxc/template [hunk 2]
```diff
@@ -1,45 +1,28 @@
...
+license="GPL-3.0-or-later, BSD-3-Clause, CC0-1.0, LGPL-2.0-only, LGPL-2.1-only,
+ LGPL-3.0-or-later, Nokia-Qt-exception-1.1, MIT BSL-1.0"
 homepage="https://keepassxc.org/"
 distfiles="https://github.com/keepassxreboot/keepassxc/releases/download/${version}/keepassxc-${version}-src.tar.xz"
-checksum=0c24a78895185b9b6b098f08a15b332c600f80aa570d608940fcc2a38e02780a
+checksum=081ff1a34da8e0e8db951b5d70824d7a2b3d299ed4ae54e6ddc1e3ac176b95a8
 
 if [ "$CROSS_BUILD" ]; then
-	hostmakedepends+=" qt5-host-tools qt5-qmake qt5-tools"
+ hostmakedepends+=" qt5-qmake"
```
Comment (maxice8) [inline srcpkgs/keepassxc/template]: wrong indent

File: srcpkgs/keepassxc/template [hunk 3]
```diff
@@ -1,45 +1,28 @@
...
-	vlicense COPYING
-	vlicense LICENSE.BOOST-1.0
-	vlicense LICENSE.BSD
-	vlicense LICENSE.CC0
-	vlicense LICENSE.NOKIA-LGPL-EXCEPTION
-	vlicense LICENSE.LGPL-3
-	vlicense LICENSE.LGPL-2.1
-	vlicense LICENSE.GPL-2
+ vlicense COPYING
+ vlicense LICENSE.BOOST-1.0
```
Comment (maxice8) [inline srcpkgs/keepassxc/template]: not required

File: srcpkgs/keepassxc/template [hunk 4]
```diff
@@ -1,45 +1,28 @@
...
+ vlicense LICENSE.BSD
+ vlicense LICENSE.CC0
```
Comment (maxice8) [inline srcpkgs/keepassxc/template]: not required

File: srcpkgs/keepassxc/template [hunk 5]
```diff
@@ -1,45 +1,28 @@
...
+ vlicense LICENSE.MIT
+ vlicense LICENSE.NOKIA-LGPL-EXCEPTION
```
Comment (maxice8) [inline srcpkgs/keepassxc/template]: not required

File: srcpkgs/keepassxc/template (hunk 1 above)
Comment (maxice8) [inline srcpkgs/keepassxc/template]: not required

**Comment by gspe** [comment]
@maxice8 I hope that indentation is ok now.
So have I to remove all the post-install license copy?

**Comment by maxice8** [comment]
On Sun, Apr 07, 2019 at 05:06:16AM -0700, Giuseppe Fierro wrote:
> @maxice8 I hope that indentation is ok now.
> So have I to remove all the post-install license copy?
> 
> -- 
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/10669#issuecomment-480584493

only the ones i marked as such

File: srcpkgs/keepassxc/template [hunk 6]
```diff
@@ -1,45 +1,28 @@
...
 
-build_options="autotype browser network sshagent yubikey"
-desc_option_autotype="Enable auto-type"
-desc_option_browser="Enable browser integration with keepassxc-browser"
-desc_option_network="Enable networking support (favicon download)"
-desc_option_sshagent="Enable SSH agent support"
-desc_option_yubikey="Enable YubiKey HMAC-SHA1 authentication support"
-build_options_default="autotype browser sshagent yubikey network"
-
 post_install() {
```
Comment (maxice8) [inline srcpkgs/keepassxc/template]: LICESE.BOOST-1.0 COPYING LICENSE.NOKIA-LGPL-EXCEPTION LICENSE.CC0 don't need to be vlicense.

File: srcpkgs/keepassxc/template [hunk 7]
```diff
@@ -1,45 +1,28 @@
...
-	$(vopt_if network libcurl-devel)
-	$(vopt_if yubikey 'libykpers-devel libyubikey-devel')"
+configure_args="-DWITH_TESTS=OFF -DWITH_XC_ALL=ON"
+makedepends="qt5-tools-devel qt5-svg-devel libgcrypt-devel libargon2-devel qrencode-devel
+	qt5-x11extras-devel libXtst-devel libykpers-devel libyubikey-devel libsodium-devel"
 short_desc="KeePassXC is a cross-platform port of “Keepass Password Safe”"
 maintainer="ibrokemypie <ibrokemypie@bastardi.net>"
-license="GPL-3.0-only"
+license="GPL-3.0-or-later, BSD-3-Clause, CC0-1.0, LGPL-2.0-only, LGPL-2.1-only,
+	LGPL-3.0-or-later, Nokia-Qt-exception-1.1, MIT BSL-1.0"
```
Comment (maxice8) [inline srcpkgs/keepassxc/template]: this indent is wrong now, inside functions is a tab, inside license= and makedepends= it is 1 space

Comment (gspe) [reply srcpkgs/keepassxc/template]: ok, I get it now

File: srcpkgs/keepassxc/template (hunk 6 above)
Comment (gspe) [reply srcpkgs/keepassxc/template]: done

**Comment by Piraty** [comment]
wtf why did you remove ALL build options??? Didn't it occur to you that people actually might use some of them?

**Pull request merged**